jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -148,9 +153,12 @@ object XGBoost extends Serializable {
    * @param missing the value represented the missing value in the dataset
    * @param featureCol the name of input column, "features" as default value
    * @param labelCol the name of output column, "label" as default value
+   * @param baseMarginCol the name of the column with the initial prediction,
+   *                      "baseMargin as default value
    * @throws ml.dmlc.xgboost4j.java.XGBoostError when the model training is failed
    * @return XGBoostModel when successful training
    */
+  // scalastyle:off
```

Comment:
Yep, it's already [there](https://github.com/dmlc/xgboost/pull/2532/files#diff-4a340c5f54c4e6a6241957fae8581075).